src/lib/database.js

```diff
@@ -565,7 +566,8 @@ const ALL_AGGREGATIONS = {
         source: "return 'WRS' + _value + '-' + "
           + "doc['properties.landsat:wrs_path'].value + "
           + "doc['properties.landsat:wrs_row'].value"
-      }
+      },
+      size: 10000,
```

Comment:
Do we have tests, but they return fewer than 10 records (so this slipped by) ?

Reply:
The return count wasn't tested (IIRC), but also our tests only populate a few items primarily to test the search querying. We should probably do something with a bit more scale to allow testing for things like this.